lib/templates.php

```diff
@@ -202,14 +203,77 @@ function gutenberg_filter_template_list_table_columns( array $columns ) {
  * @param int    $post_id     Post ID.
  */
 function gutenberg_render_template_list_table_column( $column_name, $post_id ) {
-	if ( 'slug' !== $column_name ) {
+	if ( 'slug' === $column_name ) {
+		$post = get_post( $post_id );
+		echo esc_html( $post->post_name );
+		return;
+	}
+
+	if ( 'description' === $column_name ) {
+		the_excerpt( $post_id );
+		return;
+	}
+
+	if ( 'status' === $column_name ) {
+		$post_status = get_post_status( $post_id );
+		// The auto-draft status doesn't have localized labels.
+		if ( 'auto-draft' === $post_status ) {
+			echo esc_html_x( 'Auto-Draft', 'Post status', 'gutenberg' );
+			return;
+		}
+		$post_status_object = get_post_status_object( $post_status );
+		echo esc_html( $post_status_object->label );
 		return;
 	}
-	$post = get_post( $post_id );
-	echo esc_html( $post->post_name );
 }
 add_action( 'manage_wp_template_posts_custom_column', 'gutenberg_render_template_list_table_column', 10, 2 );
 
+/**
+ * Adds the auto-draft view to the 'wp_template' post type list.
+ *
+ * @param array $views The edit views to filter.
+ */
+function gutenberg_filter_templates_edit_views( $views ) {
+	$url               = add_query_arg(
+		array(
+			'post_type'   => 'wp_template',
+			'post_status' => 'auto-draft',
+		),
+		'edit.php'
+	);
+	$is_autodraft_view = isset( $_REQUEST['post_status'] ) && 'auto-draft' === $_REQUEST['post_status'];
```

Comment:
Nit: for consistancy with `$auto_draft_view` it might be better as:
```suggestion
	$is_auto_draft_view = isset( $_REQUEST['post_status'] ) && 'auto-draft' === $_REQUEST['post_status'];
```